sdk/physical/file/file.go

```diff
@@ -225,9 +225,15 @@ func (b *FileBackend) PutInternal(ctx context.Context, entry *physical.Entry) er
 	if err := b.validatePath(entry.Key); err != nil {
 		return err
 	}
-
+cd protoc-gen-go
```

Comment:
CI indicates some syntax errors here: https://app.circleci.com/pipelines/github/hashicorp/vault/15613/workflows/39275698-fbdc-49ea-89de-ea876aa43e4d/jobs/158313 . Could you refresh my memory on what these changes are doing? 
Thanks!

Reply:
Hehe, clearly a copy paste fail.  The file change was to check whether the context was cancelled _before_ doing any file operations.  The way it was it could open the file, truncate it, then check the context, resulting in an empty file.